pom.xml

```diff
@@ -32,6 +32,33 @@
   </dependencies>
   <description>A location service for Java, identify a city, state or country by multiple terms.
   </description>
+  <modules>
+    <module>buildtools</module>
+    <module>location4j</module>
+  </modules>
```

Comment:
This is a duplicate of the modules on line 90

Reply:
Yeah, the, uh, merge didn't work quite as was desired. Fixing.